adoc/deployment-bootstrap.adoc

```diff
@@ -798,3 +798,32 @@ kube-system   kured-zv696                             1/1       Running   0
 kube-system   oidc-dex-55fc689dc-b9bxw                1/1       Running   0          2m
 kube-system   oidc-gangway-7b7fbbdbdf-ll6l8           1/1       Running   0          2m
 ----
+
+==== Co-installing multiple versions of kubectl
+
+It's possible to install multiple versions of `kubectl` on the same machine. In
+a {productname} management machine, you will be able to co-install multiple
+versions of `kubectl`, and later mark which is the version to be used with
+`update-alternatives`.
```

Comment:
You may consider mentioning that by default the kubectl with the highest version number will be selected.

Reply:
Added